interactive/admin.py

```diff
@@ -51,7 +52,9 @@ def get_readonly_fields(self, request, obj=None):
     def response_change(self, request, obj):
         if "_approve-request" in request.POST:
             obj.review(
-                request.user, timezone.now(), RegistrationRequest.ReviewStatus.APPROVED
+                request.user,
+                datetime.now(tz=timezone.utc),
+                RegistrationRequest.ReviewStatus.APPROVED,
```

Comment:
Only the `utc` alias has been deprecated, `now()` is still fine to use (and does the right thing depending on how the project is configured).

Reply:
Urgh, that's annoying. I misread a bunch of warnings. Thanks for letting me know!